src/credential.rs

```diff
@@ -213,6 +213,12 @@ impl CredentialFlat {
 
     /// Update credential fields with new values, and save
     pub fn update_from(&mut self, update_req: UpdateCredential) -> Result<(), Status> {
+        // Updating ReverseHOTP is disabled without PIN
```

Comment:
```suggestion
        // Updating ReverseHOTP is disabled
```

AFAIS you could create a ReverseHotp credential with PIN protection but would still not be able to update it.